apps/jobs/serializers.py

```diff
@@ -61,6 +65,7 @@ def to_representation(self, instance):
                 )
 
             except Exception:
+                logger.error(f"{data} didnt update failed")
```

Comment:
Logging and exception message should be the same, you can add a little detail to the logging message but it should either have same prefix or suffix as the exception message. 

For example: "Something went wrong - coudln't combine fields"